test/lib/utils/configManager.ts

```diff
@@ -10,8 +16,8 @@ test("Check config file", async (t) => {
 
 	t.deepEqual(config, {
 		ignores: [
+			"!test/sap/m/visual/Magician.spec.js",
 			"test/**/*",
```

Comment:
Usually the exclude comes first (`test/**/*`), and after that some files can be re-included (`!test/sap/m/visual/Magician.spec.js`). Note that the negation in this case does not mean "ignore", but "not ignore" (i.e. include).

The re-include in this case should not have any effect as the exclude comes afterwards.

Reply:
Done